CHANGELOG.md

```diff
@@ -10,12 +10,22 @@
 
 ### ⚠️ Breaking Changes ⚠️
 
-- `uniffi_bindgen`: Removed the `run_main` function.  It's moved to `uniffi::uniffi_bindgen_main` and now unconditionally succeeds rather than return a `Result<()>`
+- `uniffi_bindgen` no longer provides a standalone binary.  Having a standalone binary resulted in version mismatches when the `uniffi` version specified in `Cargo.toml` didn't match the `uniffi_bindgen` version installed on the system.  Read [The foreign language bindings](https://mozilla.github.io/uniffi-rs/tutorial/foreign_language_bindings.html) section of the manual for how to set up a `uniffi-bindgen` binary that's local to your workspace.
+- `uniffi_bindgen`: Removed the `run_main` function.  It's moved to `uniffi::uniffi_bindgen_main` and now unconditionally succeeds rather than return a `Result<()>`.
 
 ### What's changed
 
+- The UniFFI crate organization has been significantly reworked:
+  - Projects that use UniFFI for binding/scaffolding generation now only need to depend on the `uniffi` crate and no longer need to depend on `uniffi_bindgen`, `uniffi_build`, etc.
+  - The version numbers for each crate will no longer by kept in sync after this release.  In particular `uniffi` will have breaking changes less often than `uniffi_bindgen` and other crates.  This means that UniFFI consumers that specify their versions like `uniffi = "0.23"` will not need to bump their `uniffi` version as often as before.
 - Callback interface method calls are no longer logged (#1439)
 
+### Migrating to UniFFI 0.23
```

Comment:
I was wondering whether we could move this section to the top of the file above `<!-- next-header -->` to make it somewhat "sticky", changing the title to something like "Migrating from UniFFI 0.22"? It would probably need an extra introductory para to say something like "UniFFI 0.23 had significant changes to the versioning scheme, here's the info you need to upgrade" too.

My vague concern is that we start leveraging the new versioning scheme to make new releases far more often just to get new features someone wants to leverage. So a UniFFI user might not actually attempt to upgrade until we've made 5 new releases (with `uniffi` still on 0.23.x but the sub-crates up to 0.27.0), so this info might get lost in the noise.